### DISCUSSION TITLE
any explanation for 3times bigger file since v 0.5.8.

### DISCUSSION THREAD

**Discussion by kalle07** [janhq/jan#4104] [Get Help] [answered]
since v 0.5.8
3x times bigger file...  why ?

**Comment by A2va** [comment, marked as answer]
Previously on the first start up of Jan it downloaded the engine (llama.cpp, cuda, vulkan), now it is bundled with the app installer, but the overall size should be the same.

**Reply by louis-jan** [reply]
Thanks @A2va for the great explanation.

**Reply by kalle07** [reply]
i see   ;)
